helpers/controllers/authController.test.js

```diff
@@ -1,18 +1,26 @@
 jest.mock('bcrypt')
 jest.mock('../dbload')
+jest.mock('../mattermost')
 import bcrypt from 'bcrypt'
 import db from '../dbload'
-import { login, logout } from './authController'
+import { login, logout, signup } from './authController'
+import { chatSignUp } from '../mattermost'
 
 describe('auth controller', () => {
   let userArgs
   beforeEach(() => {
     jest.clearAllMocks()
-    userArgs = { username: 'testuser', password: 'c0d3reallyhard' }
+    userArgs = {
+      username: 'testuser',
+      password: 'c0d3reallyhard',
+      firstName: 'testuser1',
+      lastName: 'testuser1',
+      email: 'testuser@c0d3.com'
+    }
   })
 
   test('Login - should throw error when session is null', async () => {
-    expect(
+    return expect(
```

Comment:
Why is return needed here now? Just asking for my own knowledge. Thanks!

Reply:
If you're testing an async function and you're not using `await` or `return` then the test won't fail if it resolves instead of rejecting.

Reply:
This probably explains why I couldn't test something awhile back to see if it was throwing an error! lol